### PR TITLE
Fix login dialog not being visible if you open the operator screen in…

### PR DESCRIPTION
**Pull request by brian-brazil**
… P2.

### CODE REVIEW

File: html/javascript/utils.js [hunk 1]
```diff
@@ -84,8 +84,11 @@ _crgUtils = {
 			close: function() { if (callback('default'))
 				dialog.dialog("destroy");
 			}
-		});
-	},
+    });
+    // If we're in period 2, the dialog can end up off screen once all the
+    // SK rows finish loading. This ensures the dialog stays up top.
+    dialog.dialog("widget").css({"top": "5em", "position": "fixed"});
+  },
```
Comment (frank-weinberg) [inline html/javascript/utils.js]: It looks like indentation is inconsistent with the rest of the file.

Comment (brian-brazil) [reply html/javascript/utils.js]: Ah, I did the replacement the wrong way around. Fixed.

Comment (frank-weinberg) [reply html/javascript/utils.js]: The rows with brackets are still indented by spaces

Comment (brian-brazil) [reply html/javascript/utils.js]: That should do it...

**Pull request merged**